### ISSUE TITLE
Learning Rate: 3e-4

### ISSUE THREAD

**Issue by abondre98** [abondre98/Adaptive-Global-LCA-Advisor-with-Dynamic-EF-Retrieval#44]
1.	Starting Point
○	3e-4 is a typical starting LR for LoRA fine-tuning.
○	You can adjust after the first epoch if the validation loss plateaus or diverges.
2.	Optimizer Settings
Usually combined with AdamW or a similar optimizer:
 from transformers import AdamW

optimizer = AdamW(model.parameters(), lr=3e-4)
